index.html

```diff
@@ -19,12 +19,13 @@
 </head>
 <body>
     <header>
-      <div class=" container">
-        <h2 class="appTitle pd-l-75">Code Journal</h2>
+      <div class=" tabList container">
+        <h2 class=" tab appTitle pd-l-75" data-view="entry-form" >Code Journal</h2>
+        <h3 class=" tab entries" data-view="entries">Entries</h3>
```

Comment:
You should make these `a` elements, since they deal with navigation. They don't need an `href` value, but they should be `a` elements to match their semantic purpose in the HTML